kotishell/src/tokenize/tokenize_a.c

```diff
@@ -6,13 +6,31 @@
 /*   By: elenavoronin <elnvoronin@gmail.com>          +#+                     */
 /*                                                   +#+                      */
 /*   Created: 2023/06/29 16:20:23 by elenavoroni   #+#    #+#                 */
-/*   Updated: 2023/07/17 14:27:47 by elenavoroni   ########   odam.nl         */
+/*   Updated: 2023/07/20 15:14:33 by elenavoroni   ########   odam.nl         */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "tokenize.h"
 
-static void	l_tk_whitespace(t_tk_so_far *so_far, char *s)
+void	l_tk_important(t_tk_so_far *so_far, char *s)
+{
+	if (so_far->status != TK_SUCCESS)
+		return ;
+	if (so_far->token.type == TK_EMPTY)
```

Comment:
this if should be always true; let's replace it with assert

Reply:
this is not the case for all these type of functions, only for the important tokens. fixed here